README.md

```diff
@@ -5,19 +5,42 @@
 
 CleanVision automatically detects various issues in image datasets, such as images that are: (near) duplicates,
 blurry, over/under-exposed, etc. This data-centric AI package is designed as a quick first step for any computer vision
-project to find problems in your dataset, which you may want to address before applying machine learning.
+project to find problems in your dataset, which you may want to address before applying machine learning. It currently detects issues in the raw images themselves, and can be useful for any machine learning task (classification, segmentation, object detection, pose estimation, keypoint detection, image generation, etc).
+The list of all issues detected by CleanVision can be found in the table below.
+
+
+|   | Issue Type       | Description                                               | Issue Key        | Example                                      |
```

Comment:
Consider making all images in this table the same height for a more polished look

Reply:
I resized all images to same height but the near and exact duplicate images get resized when rendering and look comparitively smaller.

Reply:
ah no worries about that